.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+  },
+  extends: [
+    '@metamask/eslint-config',
+    '@metamask/eslint-config/config/nodejs',
+  ],
+  plugins: [
+    'json',
+  ],
+  parserOptions: {
+    ecmaVersion: 2018,
+  },
+  ignorePatterns: [
+    '!.eslintrc.js',
+    'node_modules/',
+  ],
+  overrides: [
+    {
+      files: ['*js'],
+      rules: {
+        'consistent-return': 'off',
+        'node/callback-return': 'off',
```

Comment:
Is adding these rules the correct approach?

Reply:
I think we should keep these rules enabled, and fix all of the no doubt many violations 😅 

If it's too painful to fix in tests, feel free to disable the rules for tests for now. Ideally we should just fix them everywhere.